src/Fields/fieldvector.jl

```diff
@@ -135,6 +135,9 @@ _deepcopy_internal(x::Field, stackdict::IdDict) =
 Base.deepcopy_internal(fv::FieldVector{T}, stackdict::IdDict) where {T} =
     FieldVector{T}(map(x -> _deepcopy_internal(x, stackdict), _values(fv)))
 
+Base.deepcopy_internal(x::Field, stackdict::IdDict) =
```

Comment:
Can we add a comment for posterity why we need this?

Reply:
Also a single test would be good, you could just test the === of spaces of deepcopied fields (and !=== their field_values)